truenas/webui
# NAS-114032 / 22.02 / NAS-114032: Center and add margins to entity-empty on entity-tables.

To test: 
- Go to a page that uses entity-table and doesn't have content.
- Make sure entity-empty is centered
